cf_cli_java_plugin.go

```diff
@@ -229,7 +249,29 @@ func (c *JavaPlugin) execute(commandExecutor cmd.CommandExecutor, uuidGenerator
 	cfSSHArguments = append(cfSSHArguments, remoteCommand)
 
 	output, err := commandExecutor.Execute(cfSSHArguments)
+	if command == heapDumpCommand {
+
+		finalFile, err := util.FindDumpFile(applicationName, heapdumpFileName)
```

Comment:
Why do we need to calculate the final heap dump file name again? Don't we have this already via `heapdumpFileName` which we constructed at line 200? Wouldn't it be sufficient if `FindDumpFile` solely checks whether the expected file has been generated or is there a situation where `heapdumpFileName != finalFile`?

Reply:
seems that jvmmon doesn't support parameters to specify the path and name for the dump generated, so added this step in case the dump was generated by jvmmon.